README.md

```diff
@@ -117,18 +117,29 @@ docker-compose up
 Database Testing
 -------
 
-Resetting The Database After Each Test
+To reset the database after each test, add:
 
 ```php
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 ```
 
-And inside each test classes.
+Also inside each test classes, add:
 
 ```php
 use DatabaseMigrations;
 ```
 
+Keep in mind that currently this isn't supported and should be removed:
+
```

Comment:
Keep in mind that these traits are not yet supported:

- `use Database Transactions;`

- `use RefreshDatabase;`

Reply:
Hi,

I've edited as your suggestion.

Thank you!